### PR TITLE
Update Akka.NET and Akka.Hosting to 1.5.0-beta6

### PR DESCRIPTION
**Pull request by eaba**

### CODE REVIEW

File: src/Akka.Persistence.Redis.Cluster.Tests/Query/RedisCurrentEventsByPersistenceIdSpec.cs [hunk 1]
```diff
@@ -36,6 +37,7 @@ public RedisCurrentEventsByPersistenceIdSpec(ITestOutputHelper output, RedisClus
             ReadJournal = Sys.ReadJournalFor<RedisReadJournal>(RedisReadJournal.Identifier);
         }
 
+        [Obsolete]
```
Comment (Arkatufus) [inline src/Akka.Persistence.Redis.Cluster.Tests/Query/RedisCurrentEventsByPersistenceIdSpec.cs]: You don't need to add these obsolete attributes

**Review by Arkatufus — CHANGES_REQUESTED** [review]
The Obsolete attributes are not nescessary

File: src/Akka.Persistence.Redis.Cluster.Tests/Query/RedisCurrentEventsByPersistenceIdSpec.cs (hunk 1 above)
Comment (eaba) [reply src/Akka.Persistence.Redis.Cluster.Tests/Query/RedisCurrentEventsByPersistenceIdSpec.cs]: ok

**Comment by Arkatufus** [comment]
The fixture class connection string is not compatible with Redis 7.x

**Review by Arkatufus — APPROVED** [review]

**Pull request merged**